modules/tinymce/src/themes/silver/main/ts/ui/toolbar/CommonToolbar.ts

```diff
@@ -39,20 +44,34 @@ export interface MoreDrawerToolbarSpec extends ToolbarSpec {
 
 export interface ToolbarGroup {
```

Comment:
I think this is more an internal type like [this one](https://github.com/tinymce/tinymce/blob/main/modules/tinymce/src/themes/silver/main/ts/ui/view/View.ts#L18) since that `AlloySpec[]`, I exported in bridge the other `ToolbarGroup` type with this commit: https://github.com/tinymce/tinymce/pull/9900/commits/3f9e15f5a3d6abb20b1bca0937a8472b5b081869

Reply:
Right I see that now.